### PR TITLE
load libGLESv3.so by default while user choose glesv3

### PR DESCRIPTION
**Pull request by bofeng-song**

### CODE REVIEW

File: native/CMakeLists.txt [hunk 1]
```diff
@@ -2383,13 +2383,10 @@ if(ANDROID)
     )
 
     find_library(LIB_EGL NAMES EGL)
-    find_library(LIB_GLESv2 NAMES GLESv2)
     target_link_libraries(cocos2d PUBLIC
         android
         OpenSLES
         ${LIB_EGL}
-        ${LIB_GLESv2}
-        GLESv2
```
Comment (minggo) [inline native/CMakeLists.txt:2392]: What will happen if want to use gles2?

File: native/cocos/renderer/gfx-gles3/GLES3Wrangler.cpp [hunk 2]
```diff
@@ -60,7 +60,7 @@ bool gles3wOpen() {
     #if __OHOS__
     libgles = dlopen("libGLESv3.so", RTLD_LAZY | RTLD_GLOBAL);
     #else
-    libgles = dlopen("libGLESv2.so", RTLD_LAZY | RTLD_GLOBAL);
+    libgles = dlopen("libGLESv3.so", RTLD_LAZY | RTLD_GLOBAL);
```
Comment (minggo) [inline native/cocos/renderer/gfx-gles3/GLES3Wrangler.cpp]: Don't need special codes for __OHOS__

File: native/CMakeLists.txt (hunk 1 above)
Comment (bofeng-song) [reply native/CMakeLists.txt:2392]: Nothing will happen, we load libGLESv2 dynamically

File: native/cocos/renderer/gfx-gles3/GLES3Wrangler.cpp (hunk 2 above)
Comment (bofeng-song) [reply native/cocos/renderer/gfx-gles3/GLES3Wrangler.cpp]: > Don't need special codes for **OHOS**

has been fixed

**Review by minggo — APPROVED** [review]

**Pull request merged**